packages/expo-updates/ios/Tests/UpdateSpec.swift

```diff
@@ -7,13 +7,13 @@ import ExpoModulesTestCore
 import EXManifests
 
 class UpdateSpec : ExpoSpec {
-  let config = try! UpdatesConfig.config(fromDictionary: [
-    UpdatesConfig.EXUpdatesConfigRuntimeVersionKey: "1",
-    UpdatesConfig.EXUpdatesConfigUpdateUrlKey: "https://exp.host/@test/test"
-  ])
-  let database = UpdatesDatabase()
-  
-  override func spec() {
+  override class func spec() {
+    let config = try! UpdatesConfig.config(fromDictionary: [
+      UpdatesConfig.EXUpdatesConfigRuntimeVersionKey: "1",
+      UpdatesConfig.EXUpdatesConfigUpdateUrlKey: "https://exp.host/@test/test"
```

Comment:
```suggestion
      UpdatesConfig.EXUpdatesConfigUpdateUrlKey: "https://u.expo.dev/00000000-0000-0000-0000-000000000000"
```

Minor but can replace classic url with EAS